mypyc/ir/func_ir.py

```diff
@@ -244,9 +248,13 @@ def format_blocks(blocks: List[BasicBlock], env: Environment) -> List[str]:
                 and ops[-1].label == blocks[i + 1]):
             # Hide the last goto if it just goes to the next basic block.
             ops = ops[:-1]
+        regex = re.compile('r[0-9]+')
```

Comment:
This needs to match whole words only, as otherwise we might be matching components of identifiers. `\b` does the trick (both at the beginning and the end).